be/src/olap/schema_change.cpp

```diff
@@ -664,7 +695,7 @@ Status SchemaChangeHandler::process_alter_tablet_v2(const TAlterTabletReqV2& req
               << ", alter_version=" << request.alter_version;
 
```

Comment:
what if _base_tablet or _new_tablet is null.

Reply:
_base_tablet and _new_tablet will be checked in process_alter_tablet.